.mergify.yml

```diff
@@ -44,7 +43,7 @@ pull_request_rules:
     actions:
       comment:
         message: This pull request is now in conflicts. Could you fix it? 🙏
-  - name: Wait for 2 days before validating merge
+  - name: Wait for some time before validating merge
```

Comment:
I came to suggest this but also maybe we can provide a custom rule which will checkout the source and perform a check for obfuscated input?! Other than that LGTM

Reply:
Please bring such ideas to https://progress.opensuse.org/issues/189363